tests/integration/components/ui-message-test.js

```diff
@@ -0,0 +1,317 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('ui-message', 'Integration | Component | ui-message', {
+  integration: true,
+});
+
+test('it renders properly', function(assert) {
+  assert.expect(2);
+
+  // Render component.
+  this.render(hbs`{{ui-message
+    class=class
```

Comment:
Property class isn't used in test, you can check additional CSS-classes through it.